lang/main-fr.json

```diff
@@ -363,19 +363,19 @@
         "videoQuality": "Accorder la qualité des appels"
     },
     "liveStreaming": {
-        "busy": "Nous travaillons sur la libération des ressources de Streaming. Veuillez réessayez dans quelques minutes.",
-        "busyTitle": "Tous les streamers sont actuellement occupés",
+        "busy": "Nous tentons de libérer des ressources de diffusion. Veuillez réessayez dans quelques minutes.",
+        "busyTitle": "Tous les flux sont actuellement occupés",
         "changeSignIn": "Changer de compte.",
-        "choose": "Choisir un flux live",
+        "choose": "Choisir un flux en direct",
         "chooseCTA": "Choisissez une option de diffusion. Vous êtes actuellement connecté comme {{email}}.",
-        "enterStreamKey": "Entrez votre clé de flux live Youtube ici",
-        "error": "Le Streaming a échoué. Veuillez réessayer.",
+        "enterStreamKey": "Entrez votre clé de diffusion Youtube Live ici",
+        "error": "La diffusion a échouée. Veuillez réessayer ultérieurement.",
         "errorAPI": "Une erreur s'est produite lors de l'accès à vos diffusions YouTube. Veuillez réessayer de vous connecter.",
-        "errorLiveStreamNotEnabled": "La diffusion en direct n'est pas activée pour  {{email}}. Merci de l'activer ou de vous connecter avec un compte où elle est déjà activée.",
+        "errorLiveStreamNotEnabled": "La diffusion en direct n'est pas activée pour {{email}}. Merci de l'activer ou de vous connecter avec un compte où elle est déjà activée.",
         "expandedOff": "La diffusion en direct a été arrêtée",
         "expandedOn": "La conférence est en cours de diffusion sur YouTube.",
         "expandedPending": "La diffusion en direct a commencé...",
-        "failedToStart": "Le Streaming n'as pas réussi à démarrer",
+        "failedToStart": "Le diffusion n'as pas réussi à démarrer",
```

Comment:
@bugeaud can you just change "Le diffusion" by "La diffusion" and ping someone for the merge request please ? :) Your work is excellent !